src/interfaces/wallet.h

```diff
@@ -146,32 +146,6 @@ class Wallet
         WalletOrderForm order_form,
         std::string& reject_reason) = 0;
 
-    //! Return whether transaction can be abandoned.
-    virtual bool transactionCanBeAbandoned(const uint256& txid) = 0;
-
-    //! Abandon transaction.
-    virtual bool abandonTransaction(const uint256& txid) = 0;
```

Comment:
What does "abandon transaction" mean exactly? Why don't we need it?

Reply:
Abandon transaction mean abandon a transaction.
Why don't we need it ? because when we do a transaction on verium, we immediatly share it with other node, and we don't have the bitcoin issue where if we don't add enough fee, nodes will not process it